saleor/graphql/webhook/types.py

```diff
@@ -22,6 +26,54 @@ def resolve_name(root: models.WebhookEvent, *_args, **_kwargs):
         return WebhookEventType.DISPLAY_LABELS.get(root.event_type) or root.event_type
 
 
+class EventDeliveryAttempt(CountableDjangoObjectType):
+    created_at = graphene.DateTime(required=True)
+    status = EventDeliveryStatusEnum(
+        description="Event delivery status.", required=True
+    )
+
+    class Meta:
+        description = "Webhook delivery attempts"
+        model = core_models.EventDeliveryAttempt
+        only_fields = [
+            "id",
+            "created_at",
+            "task_id",
+            "duration",
+            "status",
+            "response",
+            "response_headers",
+            "request_headers",
+        ]
+
+
+class EventDelivery(CountableDjangoObjectType):
+    status = EventDeliveryStatusEnum(
+        description="Event delivery status.", required=True
+    )
+    attempts = graphene.List(
+        graphene.NonNull(EventDeliveryAttempt),
+        description="Event delivery attempts.",
+        required=True,
+    )
+    event_type = WebhookEventTypeEnum(description="Webhook event type.", required=True)
+
+    class Meta:
+        description = "Event delivery."
+        model = core_models.EventDelivery
+        interfaces = [graphene.relay.Node]
+        only_fields = [
+            "id",
+            "created_at",
+            "status",
+            "event_type",
+        ]
+
+    @staticmethod
+    def resolve_attempts(root: core_models.EventDelivery, *_args, **_kwargs):
+        return core_models.EventDeliveryAttempt.objects.filter(delivery=root)
```

Comment:
You can create a data loader for this resolver.

Reply:
added.